pos_invoice_journal/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Pos Invoice Journal
+===================
+
+This module allow you to use a different journal for the invoices created 
```

Comment:
Ok, so also https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst needs to be fixed, isn'it?